modules/end-user-guide/pages/url-parameter-for-the-ide.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: CONCEPT
+:description: URL parameter for the IDE
+:keywords: ide, editor, workspace-ide, workspace-editor
+:navtitle: URL parameter for the IDE
+:page-aliases: url-parameter-for-the-in-browser-ide.adoc
+
+[id="url-parameter-for-the-ide"]
+= URL parameter for the IDE
+
+If the URL for starting a new workspace doesn't contain a parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE, which is Microsoft Visual Studio Code - Open Source.
+
+The URL parameter for specifying a supported IDE is `che-editor=`. You can use it to start a workspace with a clone of the same project repository and your choice of a supported IDE.
+
+TIP: You can use the `che-editor=` parameter when you cannot add or edit a xref:defining-a-common-ide.adoc[`/.che/che-editor.yaml`] file in the source-code Git repository to be cloned for workspaces.
+
+NOTE: If the source-code Git repository to be cloned for workspaces contains a xref:defining-a-common-ide.adoc[`/.che/che-editor.yaml`] file, then the `che-editor=` parameter overrides it.
+
+This parameter accepts two types of values:
+
+* `che-editor=__<editor_key>__`
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?che-editor=__<editor_key>__
+----
++
+
+.The URL parameter `__<editor_key>__` values for supported IDEs
+[cols="1,1a,1a"]
+|===
+| IDE | `__<editor_key>__` value | Note
+
+include::example$snip_{project-context}-table-of-supported-editors.adoc[]
+
+|===
+
+* `che-editor=__<url_to_a_file>__`
++
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?che-editor=__<url_to_a_file>__# <1>
+----
+<1> URL to a file with link:https://devfile.io/docs/2.2.0/what-is-a-devfile[devfile content].
++
+[TIP]
+====
+
+* The URL must point to the raw file content.
+
+* To use this parameter with a xref:defining-a-common-ide.adoc[`che-editor.yaml`] file, copy the file with another name or path, and remove the line with `inline` from the file.
```

Comment:
For a better visibility, we can add examples of our editors' Devfiles, like in Mario's blog [here](https://developers.redhat.com/articles/2022/07/12/how-run-vs-code-openshift-dev-spaces#).
See a small chart in the chapter `Option 2: Using the URL parameter che-editor`:
![image](https://user-images.githubusercontent.com/1636395/222507422-38dcc4ec-8223-47bd-a32a-1e326c971236.png)

Reply:
Thank you, Artem. I will have a look at it and try to think how to include this in the docs.

Reply:
@azatsarynnyy, thank you again. I will incorporate this info in a separate issue so as not to delay merging this PR. I have opened [RHDEVDOCS-5053](https://issues.redhat.com/browse/RHDEVDOCS-5053) and will work on it during the next sprint.